server/game/forcedtriggeredability.js

```diff
@@ -28,6 +28,10 @@ class ForcedTriggeredAbility extends TriggeredAbility {
         this.handler = properties.handler;
     }
 
+    isForcedAbility() {
```

Comment:
I might have gone the other way (`isThisAnAbilityDirectlyTriggeredByPlayerChoice` but with a shorter name) but this works fine. 👍